MSAL/test/unit/native_auth/public/state_machine/reset_password/ResetPasswordRequiredStateTests.swift

```diff
@@ -29,25 +29,24 @@ import XCTest
 
 final class ResetPasswordRequiredStateTests: XCTestCase {
 
-    private var correlationId: UUID!
+    private var correlationId: UUID = UUID()
```

Comment:
I thought that these variables are initialised only once per class but apparently there are initialised once per test so it's ok, UUID is different for each test so no problem here.